### DISCUSSION TITLE
HRV signal: what is default unit and how to convert my data

### DISCUSSION THREAD

**Discussion by Mario-0** [neuropsychology/NeuroKit#1000] [Q&A]
Hello, I have a bunch of Polar H10 ECG data, and the data files look like this: 

```
Phone timestamp;sensor timestamp [ns];timestamp [ms];ecg [uV]
2024-05-06T16:12:33.277;599616043400041933;0.0;8557
2024-05-06T16:12:33.284;599616043407750166;7.708233;8157
2024-05-06T16:12:33.292;599616043415458399;15.416466;8900
2024-05-06T16:12:33.300;599616043423166632;23.1247;9333
2024-05-06T16:12:33.307;599616043430874865;30.832932;9181
```

If I try the NeuroKit sample data, or I simulate an ECG signal, the format looks like this: 

`array([ 0.25706941,  0.25931073,  0.26620755, ..., -0.00104164, -0.00052164, -0.00034893])`

I want to use hrv() to compute features of my Polar data. Therefore, 2 questions: 
- What is the unit of the series created with ecg_simulate?
- Is that signal scaled from 0..1? 
- How to convert my Polar data, so they can be used as input for hrv()? 

Thank you!

**Comment by DominiqueMakowski** [comment]
The data look like a semi-colon separated file, which you can probably read with pandas (read_csv).
Once you have the data as a Pandas DataFrame, you can use NeuroKit functions on the variables.

I am not sure about the scale (but to be fair it is pretty irrelevant as far as I am concerned - and neurokit functions are typically scale independent)
